documentation/staging/content/managing-domains/accessing-the-domain/status-conditions.md

```diff
@@ -195,15 +195,22 @@ The following is a list of condition types for a Cluster resource.
   * There are no pending server shutdown requests.
 
 #### `Available` {id="cluster-available"}
-- The `status` attribute is set to  `True` when a sufficient number of WebLogic Server pods are
-  ready in the cluster. This includes either:
-  * All WebLogic Server pods in the cluster are ready, as configured in the `cluster.spec.replicas` field,
-    or if it is not configured, then in the `domain.spec.replicas` field.
-  * Some of the expected server pods for the cluster are temporarily not ready, and the number of
-    `not ready` server pods is less than or equal to `cluster.spec.maxUnavailable` which defaults to `1`.
-- The `status` attribute is also set to `True` when no WebLogic Server pods are running and this
-  is the expected state, such as when `cluster.spec.replicas` is set to `0`, or when
-  `cluster.spec.serverStartPolicy` is set to `Never`.
+- The `status` attribute is set to `True` when a sufficient number of WebLogic Server pods are
+  ready in the cluster. Both of the following must be true:
+  - At least one pod in the cluster is expected to run and is `ready`.
+  - The number of `not ready` server pods which are expected to run
```

Comment:
pods which are expected to run -> pods, which are expected to run, (two commas needed)